.github/ISSUE_TEMPLATE/technical_issue.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Technical issue
+about: Suggest an improvement users won't notice
+title: ''
+labels: 'Type: Technical issue'
```

Comment:
Not all repos have these labels. We might need to define a set of labels all repos have. Also may be worth considering reverting cht-core to using the default labels ("Type: Bug" -> "bug"). What do you think?

Reply:
I agree that we should define a list of default labels for the organization. Do we have that list at the moment? If not, [can you create it](https://docs.github.com/en/organizations/managing-organization-settings/managing-default-labels-for-repositories-in-your-organization) please and share it with me? Then I can update the default labels for the default issue types. Thanks!

Reply:
The current defaults are what was created for this repo: https://github.com/medic/cht-repo-template/labels

These do not at all match what we do in cht-core. We should standardise but to what? It would be good to be more inline with what the rest of GH does (the default set) with a few key additions where needed. We currently do some scripting on the labels (eg: https://github.com/medic/cht-core/blob/master/scripts/release-notes/index.js ) but nothing that would be difficult to change, and standardising would mean the scripts are usable across repos.

In order to unblock this PR, let's use the default labels linked above. Then new repos will be internally consistent.